SparkPost/2web2ui
# FE-1293 - Fix Messaging Usage chart data

Note: Give your PR a short title. Something like "_TICKET NUMBER: feature description_" is good.

### What Changed

- Fixes a bug where the gradient (blue to yellow) for Messaging Usage chart was being calculated based on the filled data rather than the raw data

### How To Test

- Run the app
- Verify the Messaging usage chart changes from blue to yellow when crossing the usage threshold

### To Do

  > Manage reviewers' expectations on missing features, tests, bugs, late design changes, ...)
